src/analyzer/model.ts

```diff
@@ -36,10 +37,11 @@ export interface TruckConfig {
   unionedAuthors: string[][]
   hiddenFiles: string[]
   invalidateCache: boolean
+  colorSeed?: string
```

Comment:
seeds are usually numbers. why is this a string?

Reply:
It seems that the library `uniqolor` takes either a UUID or anything that can be converted to a string or number, so I think using a string is fine?

Reply:
yes. I saw that after!